applications/crossbar/src/modules/cb_cdrs.erl

```diff
@@ -270,7 +270,7 @@ validate(Context, _, _) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec load_cdr_summary(cb_context:context(), req_nouns()) -> cb_context:context().
-load_cdr_summary(Context, [_, {?KZ_ACCOUNTS_DB, [_]} | _]) ->
+load_cdr_summary(Context, [_, {?KZ_ACCOUNTS_DB, _} | _]) ->
```

Comment:
When would this not be a list of one? `/v2/accounts/{ACCOUNT_ID}/cdrs` would give you `[{<<"cdrs">>, []}, {<<"accounts">>, [<<"{ACCOUNT_ID}">>]}]`. What request is coming in with more parameters to the `accounts` endpoint?

Reply:
There is no such a request, so basically `[_] =:= _` and also `[_]` is 3 characters to type compare to `_`.